docs/API.md

```diff
@@ -301,8 +341,223 @@ Rejects a pending invite for the authenticating account
 await api.rejectRealmInvitation('1234567')
 ```
 
+<details>
+<summary>Output</summary>
+
+No output
+
+</details>
+
+---
+
+### removePlayerFromRealm
+
+(realmId: string, xuid: string) => Promise\<Realm>
```

Comment:
Why specifically return a Realm here?

Reply:
In the bedrock Realm API docs, it states it returns all information about the Realm. Should we not be returning all the Realm's information as states in the API docs?

Reply:
Nevermind, ignore that. I understand now. Thank you